src/winforms/toga_winforms/app.py

```diff
@@ -144,6 +146,7 @@ def run_app(self):
             self.create()
 
             self.native.ThreadException += self.winforms_thread_exception
+            self.interface.main_window._impl.native.KeyUp += self.winforms_key_press
```

Comment:
Two questions here:
1) Is keyUp the right signal to catch? Why catch the release, not the press (i.e., keyDown)?
2) The code placement is odd to me. This doesn't affect how the code runs, but putting a keypress hander on the window in between two native app handlers seems out of place.